config/crd/bases/infrastructure.cluster.x-k8s.io_azuremanagedmachinepools.yaml

```diff
@@ -42,6 +42,13 @@ spec:
             description: AzureManagedMachinePoolSpec defines the desired state of
               AzureManagedMachinePool.
             properties:
+              mode:
+                description: 'Mode - represents mode of an agent pool. Possible values
+                  include: ''System'', ''User'''
```

Comment:
nit: the quotes came out a bit weird here. maybe just drop the single quotes around the values themselves? or can we escape them better?

Reply:
ack